cmd/unifibackup/main.go

```diff
@@ -32,73 +32,56 @@ var (
 	})
 )
 
+var CLI struct {
```

Comment:
Could this be made a local variable within `main()`, or at least be unexported?

Reply:
yea. Their getting started docs are a little rough, so i didn't realize the name didn't matter till much later.